lightning/src/ln/channel.rs

```diff
@@ -4317,7 +4323,7 @@ impl<Signer: Sign> Channel<Signer> {
 	}
 
 	pub fn channel_update(&mut self, msg: &msgs::ChannelUpdate) -> Result<(), ChannelError> {
-		let usable_channel_value_msat = (self.channel_value_satoshis - self.counterparty_selected_channel_reserve_satoshis) * 1000;
+		let usable_channel_value_msat = (self.channel_value_satoshis - self.counterparty_selected_channel_reserve_satoshis.unwrap()) * 1000;
```

Comment:
Is it possible for our counterparty to send send a `channel_update` before we're ready for this `unwrap`?

Reply:
Grrrrr, yes, you're totally right. Further, I don't think we should really be rejecting `channel_update`s based on the reserve - I mean, sure, they shouldn't include it, but a node could reasonably just be ignoring it and we shouldn't close the channel for that.